modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -217,21 +217,22 @@ void NetworkedMultiplayerENet::poll() {
 
 	_pop_current_packet();
 
+	if (!host || !active) // Might be disconnected
```

Comment:
`if`s should always include curly brackets

Reply:
fixed